docs/tutorial/quick-start.rst

```diff
@@ -106,45 +65,24 @@ more in-depth installation instructions, refer to :ref:`rocm-install-overview`.
+                        sudo zypper addrepo https://download.opensuse.org/repositories/devel:languages:perl/{{ os_version}}/devel:languages:perl.repo
                 {% endif %}
-                                sudo zypper install kernel-default-devel
-                                sudo usermod -a -G render,video $LOGNAME # Adding current user to Video, Render groups. See prerequisites.
-                                sudo zypper --no-gpg-checks install https://repo.radeon.com/amdgpu-install/|amdgpu_version|/sle/{{ os_version }}/amdgpu-install-|amdgpu_install_version|.noarch.rpm
-                                sudo zypper refresh
-                                sudo zypper install amdgpu-dkms
-                                sudo zypper install rocm
-                                echo "Please reboot system for all settings to take effect."
-
-                        .. tab-item:: AMDGPU installer
-                            :sync: amdgpu-installer
-
-                            .. code-block:: bash
-                                :substitutions:
-
-                {% if os_version == "15.4" %}
-                                # Installing Perl module from SLES 15.5, as it was removed from 15.4
-                                sudo zypper addrepo https://download.opensuse.org/repositories/devel:/languages:/perl/15.5/devel:languages:perl.repo
-                {% else %}
-                                sudo zypper addrepo https://download.opensuse.org/repositories/devel:languages:perl/{{ os_version}}/devel:languages:perl.repo
-                {% endif %}
-                                sudo zypper install kernel-default-devel
-                                sudo usermod -a -G render,video $LOGNAME # Adding current user to Video, Render groups. See prerequisites.
-                                sudo zypper --no-gpg-checks install https://repo.radeon.com/amdgpu-install/|amdgpu_version|/sle/{{ os_version }}/amdgpu-install-|amdgpu_install_version|.noarch.rpm
-                                sudo amdgpu-install --usecase=graphics,rocm
+                        sudo zypper install kernel-default-devel
+                        sudo usermod -a -G render,video $LOGNAME # Add the current user to the render and video groups
+                        sudo zypper --no-gpg-checks install https://repo.radeon.com/amdgpu-install/|amdgpu_version|/sle/{{ os_version }}/amdgpu-install-|amdgpu_install_version|.noarch.rpm
+                        sudo zypper refresh
+                        sudo zypper install amdgpu-dkms rocm
                 {% endfor %}
 
+.. note::
+
+   To apply all settings, reboot your system.
 
-If you encounter install issues, you can refer to the :doc:`troubleshooting <../how-to/native-install/install-faq>` page.
+If you have issues with your installation, see :doc:`Troubleshooting <../how-to/native-install/install-faq>`.
```

Comment:
New paragraph to add at bottom - link to the post install page.

After completing the installation, review the **post-installation instructions**.

Reply:
Added in https://github.com/ROCm/rocm-install-on-linux/pull/176